### DISCUSSION TITLE
StokesFlow Algorithm Yielding Variable Results

### DISCUSSION THREAD

**Discussion by falquet** [PMEAL/OpenPNM#2927] [Q&A] [answered]
Hello,

I am using the StokesFlow function after generating a pore network with the Snow2 algorithm to calculate the permeability of a fractured crust. However, I observe that the calculated permeability varies when I rerun the simulation under identical conditions. Is this behavior expected ?

Thank you.

**Comment by jgostick** [comment, marked as answer]
It is possible that you are not removing the 'pore.diameter' and 'throat.diameter' models first, so that pore/throat sizes are being overwritten.

**Reply by falquet** [reply]
Thank you for your response ! The problem was that I forgot to include these lines : 

    pn['pore.diameter'] = pn['pore.equivalent_diameter']
    pn['throat.diameter'] = pn['throat.inscribed_diameter']
    pn['throat.spacing'] = pn['throat.total_length'].

Thanks again !
